.gitignore

```diff
@@ -19,4 +19,5 @@ packages/**/*.tgz
 packages/**/package-lock.json
 tmp
 # Verdaccio storage path
-storage
\ No newline at end of file
+storage
+.ackrc
```

Comment:
I had to add an .ackrc to the compass folder so I can use [ack](https://beyondgrep.com/) to search for code and ignore all the giant packages/*/lib/index.js files.